src/Cron_Event_Command.php

```diff
@@ -275,17 +278,17 @@ public function run( $args, $assoc_args ) {
 	protected static function run_event( stdClass $event ) {
 
 		if ( ! defined( 'DOING_CRON' ) ) {
-			define( 'DOING_CRON', true );
+			define( 'DOING_CRON', true ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound
```

Comment:
Please add a comment stating why this is whitelisted.
```suggestion
			define( 'DOING_CRON', true ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound -- Making sure a WP native constant is available.
```